actions/orders-monitor/action.yaml

```
@@ -0,0 +1,65 @@
+  using: 'composite'
+  steps:
+    - name: Replace orders by name
+      shell: bash
+      run: |
+        set -e
+        URL="${{ inputs.url }}"
+        API_KEY="${{ inputs.api_key }}"
+        ORDERS='${{ inputs.orders }}'
+        NAME='${{ inputs.name }}'
+        METADATA='${{ inputs.metadata }}'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)